com.woltlab.wcf/templates/uploadFieldComponent.tpl

```diff
@@ -12,7 +12,13 @@
 		<ul class="formUploadHandlerList" id="{$uploadFieldId}uploadFileList" data-internal-id="{$uploadField->getInternalId()}">
 			{foreach from=$uploadFieldFiles item=file}
 				<li class="box64 uploadedFile" data-unique-file-id="{$file->getUniqueFileId()}">
-					<span class="icon icon64 fa-{$file->getIconName()}"></span>
+					{if $file->isImage()}
+						<a href="{$file->getImage()}" class="jsImageViewer">
+							<img src="{$file->getImage()}" loading="lazy" alt="" class="formUploadHandlerContentListImage">
```

Comment:
`loading="lazy"` implicitly requires you to provide the explicit dimensions using `height="…"` and `width="…"`.